drf_braces/serializers/enforce_validation_serializer.py

```diff
@@ -129,7 +143,7 @@ def _create_enforce_validation_serializer(serializer, strict_mode_by_default=Tru
         elif isinstance(field, serializers.Field):
             replacement = add_base_class_to_instance(
                 field,
-                EnforceValidationFieldMixin,
+                validation_serializer_field_mixin_class,
```

Comment:
every recursive call to `_create_enforce_validation_serializer` should pass the mixin arg now as well

Reply:
🤦‍♂️